src/main/java/com/comcast/zucchini/AbstractZucchiniTest.java

```diff
@@ -264,7 +264,13 @@ else if(result.isJsonObject()) {
                 }
             }
 
-            cleanup(context);
+            try {
+                cleanup(context);
+            }
+            catch(Throwable t) {
+                LOGGER.error("ERROR: {}", t);
```

Comment:
since we have no implementation of cleanup, I don't think it's wise for us to catch exceptions that have been introduced by the client.